packages/dds/tree/src/simple-tree/toFlexSchema.ts

```diff
@@ -56,21 +60,25 @@ import { TreeConfiguration } from "./tree.js";
 function cursorFromUnhydratedRoot(
 	schema: ImplicitFieldSchema,
 	tree: InsertableTreeNodeFromImplicitAllowedTypes,
+	schemaAndPolicy: SchemaAndPolicy | undefined = undefined,
 ): ITreeCursorSynchronous {
 	const data = extractFactoryContent(tree as InsertableContent);
 	const normalizedFieldSchema = normalizeFieldSchema(schema);
 	return (
-		cursorFromNodeData(data, normalizedFieldSchema.allowedTypes) ??
+		cursorFromNodeData(data, normalizedFieldSchema.allowedTypes, schemaAndPolicy) ??
 		fail("failed to decode tree")
 	);
 }
 
-export function toFlexConfig(config: TreeConfiguration): TreeContent {
+export function toFlexConfig(
+	config: TreeConfiguration,
+	schemaAndPolicy: SchemaAndPolicy | undefined = undefined,
```

Comment:
We should probably add some basic docs to this function. For now, could you add a `@param` comment explaining `schemaAndPolicy` - in particular, what it means for it to be undefined.